content-test/meta/shims/EventEmitter.test.js

```diff
@@ -0,0 +1,24 @@
+const EventEmitter = require("shims/_utils/EventEmitter");
+const noop = () => {};
+
+describe("EventEmitter shim", () => {
```

Comment:
nit: just for consistency with other shim tests, can you drop the "shim" in the name?